src/homework26/Homework.java

```diff
@@ -0,0 +1,17 @@
+package homework26;
+
+public class Homework {
+    int id;
```

Comment:
I understand that getter and setter methods are lazy to implement, but having fields with direct access is more dangerous. Please use `private` for fields and implement getters and setters (or use lombok).